bundles/org.openhab.core.io.rest/src/main/java/org/eclipse/smarthome/io/rest/JSONResponse.java

```diff
@@ -74,8 +74,8 @@ public static Response createErrorResponse(Response.StatusType status, String er
      * Depending on the status, create a Response object containing either the entity alone or an error JSON
      * which might hold the entity as well.
      *
-     * @param status the status for the response.
-     * @param entity the entity which is transformed into a JSON stream.
+     * @param status       the status for the response.
```

Comment:
Some minor formatting stuff. Maybe this https://github.com/eclipse/smarthome/issues/6424#issuecomment-471728524 helps.

Reply:
This has been done by the official IDE, we should check that setup (and yes, it should be fixed here, too)

Reply:
Done